src/types/cronjob.ts

```diff
@@ -0,0 +1,30 @@
+import { K8sResourceCommon } from './k8s';
+
+export type CronJob = K8sResourceCommon & {
```

Comment:
follow Naming convention for K8s Resources
```suggestion
export type CronJobKind = K8sResourceCommon & {
```